examples/missions/example.json

```diff
@@ -1,91 +1,61 @@
-[
-  {
-    "faction": "federation",
-    "location": {
-      "$ref": "reference to system json",
-      "planet": "Andoria prime"
-    },
-    "mission_title": "Beneath the Ice",
-    "mission_type": "start",
-    "next_mission": {
-      "$ref": "reference to mission."
-    },
-    "objectives": [
-      {
-        "title": "Meet the Romulan Informant on Moya.",
-        "main_objective": {
-          "$ref": "#objectives/items"
-        },
-        "obj_id": 1,
-        "task": [
-          {
-            "$ref": "reference to system json",
-            "planet": "Moya",
-            "task_type": "travel"
-          },
-          {
-            "$ref": "reference to system json",
-            "task_title": "Destroy Suliban Interceptor",
-            "task_type": "destroy"
-          },
-          {
-            "$ref": "reference to system json",
-            "task_title": "Destroy Suliban Interceptor",
-            "task_type": "kill"
-          },
-          {
-            "resource": "dilithium",
-            "task_type": "deliver",
-            "value": 300
-          },
-          {
-            "options": [
-              {
-                "option_id": 1,
-                "option_title": "smuggle Garan across...",
-                "rewards": {
-                  "$ref": "reference"
-                }
-              }
-            ],
-            "task_type": "choose"
-          },
-          {
-            "task_title": "Return to your station",
-            "task_type": "return"
-          }
-        ]
-      }
-    ],
-    "previous_mission": {
-      "$ref": "reference to mission."
-    },
-    "recommended_ops": 27,
-    "rewards": [
-      {
+{
+  "arc": "borg arc 4",
+  "faction": "romulan",
+  "locations": [
+    {
+      "$ref": "systems/elvros_47",
+      "planet": "strinax"
+    }
+  ],
+  "mission_name": "A Force of Nature",
+  "mission_type": "start",
+  "next_missions": [
+    {
+      "$ref": "missions/honor_for_three",
+      "option": 1
+    }
+  ],
+  "objectives": [
+    {
+      "title": "Bring Spock to Renon.",
+      "id": 1,
+      "rewards": {
         "materials": [
           {
             "type": "gas",
             "grade": 3,
             "rarity": "uncommon",
-            "value": 50
+            "value": 18
           }
         ],
         "others": [
           {
-            "type": "federation credits",
-            "value": 330
+            "type": "romulan points",
```

Comment:
The correct name is 'reputation'

Reply:
The info was based on lcars data. 'Reputation' is the goal you reach like Respected and Ally. The points itself are called Faction Reputation Points, in game they also use the 'points': https://cdn.discordapp.com/attachments/869657307955490817/886591521435041822/Screenshot_20210912_143717_com.scopely.startrek.jpg

Reply:
Ok, this has been retranslated! 👍